install.sh

```diff
@@ -77,26 +77,49 @@ process_arguments() {
 verify_configuration_file() {
   log "Verifying the config file."
 
+  HAS_VALIDATION_ERR=false
```

Comment:
@yzha645 you need to make this `HAS_VALIDATION_ERR=` as discussed over Slack, let's use the `1` approach.

Reply:
Good spot. Thank you! Fixed.